packages/logging/src/servicies/logger/logger.service.ts

```diff
@@ -2,8 +2,6 @@ import { context, trace } from "@opentelemetry/api";
 import { isHttpError } from "http-errors";
 import pino, { Bindings, Logger as PinoLogger, LoggerOptions } from "pino";
 import { gcpLogOptions } from "pino-cloud-logging";
-import pinoFluentd from "pino-fluentd";
-import pretty from "pino-pretty";
```

Comment:
this is still needed, this is for readability when developing locally

Reply:
actually, `pinoFluentd ` and `pretty` is not working on client side @ygrishajev

Reply:
well, this was the point of the other task to make it work nicely in both envs.
Basically we need to load pretty conditionally, or in the worst case scenario have 2 loggers.

Reply:
i see from the doc it's possible to use it without import https://github.com/pinojs/pino-pretty?tab=readme-ov-file#programmatic-integration

meaning we can add this option conditionally based on the runtime env